code-for-charlottesville/housinghub
# Backend scaffolding

## Changes

- create basic flask backend
- template out methods we will need
- add relevant testing
- misc admin and configuration

## How to test

Follow along each section of the readme, make sure you are able to build, test, and understand the project
